<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_log_in">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>&lt; 매수 &gt;</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>총</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>호가</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>원</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>주</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_call_">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>매수</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_call_vol">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_call_price">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>&lt; 매도 &gt;</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>총</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>호가</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>주</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>원</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_put_">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>매도</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_put_vol">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_put_price">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="ck_put_all">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>전체</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 종목</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>20</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lb_cur_code">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_int_code">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>151</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lb_price">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>현재가</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>1161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>정지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_real_data">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>시스템 트레이딩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_basic_data">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>주식 기본 정보</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>20</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="list_log">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>251</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_acc_detail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>503</y>
      <width>871</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>192</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>941</width>
      <height>192</height>
     </size>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>종목코드</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>175</red>
       <green>175</green>
       <blue>175</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>종목명</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>175</red>
       <green>175</green>
       <blue>175</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>평가손익</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>수익률</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>매입가</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>현재가</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>매입금액</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>평가금액</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>보유수량</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>가능수량</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>175</red>
       <green>175</green>
       <blue>175</blue>
      </color>
     </property>
    </column>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>139</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>트레이딩 시작</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>139</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>트레이딩 종료</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_order_log">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>251</width>
      <height>391</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>                  &lt;&lt; 주문 기록 &gt;&gt;</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_auto">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>예약</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_acc1">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>440</y>
      <width>151</width>
      <height>62</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>총 매입</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>총 평가</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tb_acc2">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>440</y>
      <width>151</width>
      <height>62</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>평가 손익</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>수익률</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tb_acc4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>440</y>
      <width>151</width>
      <height>62</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>부가금</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>실현 손익</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_ref_acc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌 조회</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_acc3">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>440</y>
      <width>151</width>
      <height>62</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>총 매수</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>총 매도</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cb_acc">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌 번호</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>btn_log_in</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
